### PR TITLE
Add Tests For Student Usage

### PR DESCRIPTION
**Pull request by zkagin**
Adds two tests for student usage, checking that it generally functions and that the date window works.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ class Config(object):
 
 class TestConfig(Config):
     DB_TYPE = "sqlite"
-    DEBUG = False
+    DEBUG = True
```
Comment (dchess) [inline config.py:109]: @zkagin Where is the debug logging being used in testing?

File: api.py [hunk 2]
```diff
@@ -303,18 +302,13 @@ def preprocess_records(self, records):
             row = {}
             row["Email"] = record.get("entity").get("userEmail")
             row["AsOfDate"] = record.get("date")
-            row["LastUsedTime"] = self._parse_classroom_last_used(
-                record.get("parameters")
-            )
-            row["ImportDate"] = datetime.today().strftime("%Y-%m-%d")
```
Comment (dchess) [inline api.py:309]: @zkagin Why are you removing ImportDate as well?

File: config.py (hunk 1 above)
Comment (zkagin) [reply config.py:109]: Just to suppress retry requests when a test fails. The logging doesn't actually print out in the tests anyways.

File: api.py (hunk 2 above)
Comment (zkagin) [reply api.py:309]: From `self.columns`, it doesn't look like this is being used at all. Was the intent that this should be a column?

Comment (dchess) [reply api.py:309]: @zkagin Yes. You caught a bug I hadn't flagged yet. That should be a derived column. It was populating blank in prod but hadn't been added as an issue yet.

Comment (zkagin) [reply api.py:309]: Got it, fixed!

**Comment by zkagin** [comment]
@dchess Changes made, take a look!

**Pull request merged**